elasticdl/python/worker/worker.py

```diff
@@ -335,6 +376,17 @@ def report_variable(self):
         for ps_id in range(len(self._ps_stubs)):
             self.report_variable_to_ps(ps_id)
 
+    def collect_edl_embedding_name_values(self):
+        embedding_name_values = []
+        for layer in self._embedding_layers:
+            embedding_name_values.append((layer.name, layer.embedding_and_ids))
+        for column in self._embedding_columns:
+            embedding_name_values.append(
+                (column.name, column.embedding_and_ids)
+            )
```

Comment:
May `layer.name` be the same as `column.name` ?

Reply:
The name format for column.name is {column_field}_embedding_elasticdl, the layer name is user defined. I think the probability is low for name conflict.

Reply:
Whether do we need to check there are name conflicts?

Reply:
Good suggestion, added the check logic